packages/helm-charts/common/templates/_helpers.tpl

```diff
@@ -0,0 +1,30 @@
+{{- define "common.init-genesis-container" -}}
```

Comment:
Since we have the static nodes on GCP, should we just pull them into their own init container as well to share?